core/services/ardupilot_manager/main.py

```diff
@@ -1,6 +1,50 @@
 #! /usr/bin/env python3
+import json
+from typing import Any, Dict, List
+
+import uvicorn
 from ArduPilotManager import ArduPilotManager
+from fastapi import Body, FastAPI
+from mavlink_proxy.Endpoint import Endpoint
+from starlette.responses import Response
+
+
+class PrettyJSONResponse(Response):
+    media_type = "application/json"
+
+    def render(self, content: Any) -> bytes:
+        return json.dumps(
+            content,
+            ensure_ascii=False,
+            allow_nan=False,
+            indent=2,
+            separators=(", ", ": "),
+        ).encode("utf-8")
+
+
+app = FastAPI(default_response_class=PrettyJSONResponse)
+autopilot = ArduPilotManager()
+
+
+@app.get("/endpoints", response_model=List[Dict[str, Any]])
+def get_available_endpoints() -> Any:
+    return [endpoint.asdict() for endpoint in autopilot.get_endpoints()]
+
+
+@app.post("/endpoint", response_model=bool)
+def create_new_endpoint(endpoint: Endpoint = Body(...)) -> Any:
+    if autopilot.add_endpoint(endpoint):
+        return True
+    return False
+
+
+@app.delete("/endpoint", response_model=bool)
+def remove_endpoint(endpoint: Endpoint = Body(...)) -> Any:
+    if autopilot.remove_endpoint(endpoint):
```

Comment:
`return autopilot.remove_endpoint(endpoint)`

Reply:
I wonder how I still do that...